flanker/addresslib/address.py

```diff
@@ -390,27 +547,61 @@ def supports_routing(self):
 
     def full_spec(self):
```

Comment:
BTW, not trying to be difficult here. Just seemed like the correct behavior so I thought I'd make my case :)